tests/query_engines/test_mssql_lib.py

```diff
@@ -213,10 +209,3 @@ def list_all(self):
             return sorted(
                 row[0] for row in conn.execute(query) if row[0].startswith(self._prefix)
             )
-
-    def drop_all(self):
-        tables = self.list_all()
-        with self.engine.connect() as conn:
-            for table in tables:
-                conn.execute(sqlalchemy.text(f"DROP TABLE {self.database}..{table}"))
-            conn.commit()
```

Comment:
why don't we need to drop the db tables?

Reply:
I *think* because the tests all do `_populate_table` which drops the table if it exists and recreates it

Reply:
We don't need the drop tables if all the code under test works correctly and cleans up after itself. But if the tests fail then we'll leave detritus around. This is a general issue, I think, with attempting to enforce coverage of code which is only there to clean up if something goes wrong in a test: under normal circumstances it won't get executed, but we still want it there.